common/test/acceptance/tests/helpers.py

```diff
@@ -263,6 +247,41 @@ def get_modal_alert(browser):
     return browser.switch_to.alert
 
 
+class EventsTestMixin(object):
+    """
+    Helpers and setup for running tests that evaluate events emitted
+    """
+    def setUp(self):
+        super(EventsTestMixin, self).setUp()
+        self.cleanEvents()
+        self.event_collection = MongoClient()["test"]["events"]
+        self.start_time = datetime.now()
```

Comment:
You might need to shuffle this around to get the tests to pass. Looks like `cleanEvents` is called before an `event_collection` is defined.